### PR TITLE
Wallet Desktop App via Tauri + Menu

### PR DESCRIPTION
**Pull request by hariria**
## Motivation

With [Tauri](https://tauri.studio/), I was able to successfully create a native Mac apple silicon desktop app for the wallet. We are also able to do the same for Linux and Windows Desktop apps which is very exciting!

### Capabilities
- [x] It has feature parity with the chrome extension (except for dApp support ofc)
- [x] Stores state locally
- [x] Able to communicate with an endpoint, make requests
- [x] Implemented Menu with ability to copy paste, more functionality can be added later
- [x] Compiles to `.app` or `.dmg` for Mac users, can also compile to executables for Linux and Windows if needed

### Further steps
- [ ] Get an Apple Developer Account in order to sign and distribute `.dmg`s

## Test Plan

https://user-images.githubusercontent.com/31301117/172195923-83117056-f0f8-4519-a1b8-7a8d5868156e.mov

### CODE REVIEW

File: ecosystem/web-wallet/src-tauri/src/menu.rs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            .add_native_item(MenuItem::SelectAll),
+    );
+
+    let window_menu = Submenu::new(
+        "Window",
+        Menu::new()
+            .add_native_item(MenuItem::Minimize)
+            .add_native_item(MenuItem::Zoom),
+    );
+
```
Comment (kent-white) [inline ecosystem/web-wallet/src-tauri/src/menu.rs:35]: These seems unnecessary?

File: ecosystem/web-wallet/package.json [hunk 2]
```diff
@@ -31,8 +31,10 @@
     "start": "react-scripts start",
     "lint": "eslint ./src",
     "lint:fix": "eslint ./src --fix",
+    "tauri": "tauri",
```
Comment (kent-white) [inline ecosystem/web-wallet/package.json:34]: What does this do compared to the build:tauri?

**Review by kent-white — APPROVED** [review]

File: ecosystem/web-wallet/package.json (hunk 2 above)
Comment (hariria) [reply ecosystem/web-wallet/package.json:34]: I can get rid of it if you don't think it's necessary @kent-white ?

**Pull request closed without merging**